CHANGELOG.md

```diff
@@ -2,6 +2,113 @@
 
 All notable changes to this project will be documented in this file.
 
+## Unreleased - as of Sprint 45 end 2016-08-22
+
+### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+45+Ending+Aug+22%2C+2016%22+label%3Aenhancement)
+
+- Automate
+  - Enhanced messaging for provisioning: Displayed elements
+    - ManageIQ Server name
+    - Name of VM/Service being provisioned
+    - Current Automate state machine step
+    - Status message
+    - Provision Task Message
+    - Retry count (when applicable)
+  - New method `taggable?` to programmatically determine if a Service Model class or instance is taggable.
+  - Generic Objects: Model updates
+    - Associations
+    - Tagging
+    - Service Methods: `add_to_service / remove_from_service`
+  - Git Automate support
+    - Branch/Tag support
+    - Contents are locked,and can be copied to other domains for editing
+    - Editable properties
+      - Enabled/Disabled
+      - Priority
+      - Removal of Domain
+    - Dedicated Server Role to store the repository
+- Platform
+  - PostgreSQL High Availability
+    - Primary/Standby DB config in Appliance Console
+    - Database-only appliance config in Appliance Console
+    - Failover Monitor
+  - Tenancy
+    - Groundwork in preparation for supporting multiple entitlements
+    - ApplicationHelper#role_allows and User#role_allows? combined and moved to Rbac
+    - Post refresh hook to queue mapping of Cloud Tenants
+  - Database maintenance scripts added to appliance
+- Providers
+  - Containers: Models for container deployments
+  - Google
+    - Preemptible Instances
+    - Retirement support
+  - Hawkular    
+    - Alerts
+       - Link miq alerts and hawkular events on the provider
+       - Convert miq alerts/profiles to hawkular group triggers/members of group triggers
+       - Sync the provider when miq alerts and alert profiles are created/updated
+   - Added entities: Domains and Server Groups
+       - Including their visualization in topology
+       - Limitation : still missing relevant icons that need to be designed by uxd
+   - Datasource entity now has deletion operation of the
+    - Support more event types for datasource and deployment
```

Comment:
The indent here doesn't align right, so not sure if it should be outdented one space, or indented one more space.